Remove support for SVGTRefElement

lukewarlow opened the pull request:
#### c388503a416e404807f4eba1a3c2ad131c003bc7
<pre>
Remove support for SVGTRefElement
<a href="https://bugs.webkit.org/show_bug.cgi?id=246191">https://bugs.webkit.org/show_bug.cgi?id=246191</a>

Reviewed by NOBODY (OOPS!).

This patch removes SVGTRefElement and associated tests.

WebKit is the only engine that supports it, and it&apos;s not part of the SVG spec anymore.

* LayoutTests/fast/block/child-not-removed-from-parent-lineboxes-crash.html:
* LayoutTests/fast/rendering/nested-render-tree-update-crash-expected.txt: Removed.
* LayoutTests/fast/rendering/nested-render-tree-update-crash.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-change-in-shadow-tree-expected.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-change-in-shadow-tree.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-in-shadow-tree-expected.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-in-shadow-tree.html: Removed.
* LayoutTests/http/tests/svg/resources/svg-tref.svg: Removed.
* LayoutTests/http/tests/svg/tref-adoptNode-crash-expected.txt: Removed.
* LayoutTests/http/tests/svg/tref-adoptNode-crash.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/accessibility/crashtests/serialize-with-no-document.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/idlharness.window.js:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/tools/third_party/html5lib/html5lib/filters/sanitizer.py:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/glib/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/glib/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/glib/svg/custom/text-linking-expected.txt:
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/glib/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/text-align-04-b-expected.txt:
* LayoutTests/platform/glib/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/gtk/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/gtk/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/ios/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/ios/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/text/text-align-04-b-expected.txt:
* LayoutTests/platform/ios/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/ios/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/mac/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/mac/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/mac/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/mac/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/svg/W3C-SVG-1.1/resources/text-align-02-f.svg:
* LayoutTests/svg/W3C-SVG-1.1/resources/text-extTref-BE-18-targ.svg: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-align-04-b.svg:
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b-w3c.png: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b.svg: Removed.
* LayoutTests/svg/animations/animate-update-crash-expected.txt: Removed.
* LayoutTests/svg/animations/animate-update-crash.xhtml: Removed.
* LayoutTests/svg/batik/text/textProperties-expected.txt:
* LayoutTests/svg/batik/text/textProperties.svg:
* LayoutTests/svg/batik/text/textStyles.svg:
* LayoutTests/svg/custom/broken-internal-references.svg:
* LayoutTests/svg/custom/bug78700-expected.txt: Removed.
* LayoutTests/svg/custom/bug78700.svg: Removed.
* LayoutTests/svg/custom/disallowed-text-content-rendering.svg:
* LayoutTests/svg/custom/global-constructors-expected.txt:
* LayoutTests/svg/custom/global-constructors.html:
* LayoutTests/svg/custom/invalid-ctm.svg:
* LayoutTests/svg/custom/invalid-text-content-expected.txt:
* LayoutTests/svg/custom/invalid-text-content.svg:
* LayoutTests/svg/custom/remove-subtree-including-text-with-duplicate-id-referenced-by-later-tref-expected.txt: Removed.
* LayoutTests/svg/custom/remove-subtree-including-text-with-duplicate-id-referenced-by-later-tref.html: Removed.
* LayoutTests/svg/custom/reparent-tref-element-expected.txt: Removed.
* LayoutTests/svg/custom/reparent-tref-element.html: Removed.
* LayoutTests/svg/custom/text-linking.svg:
* LayoutTests/svg/custom/text-tref-03-b-change-href-dom.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-change-href.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic-expected.txt: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic2-expected.txt: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic2.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-referenced-element-removal.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-tref-removal.svg: Removed.
* LayoutTests/svg/custom/tref-clone-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-clone-crash.html: Removed.
* LayoutTests/svg/custom/tref-nested-events-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-nested-events-crash.svg: Removed.
* LayoutTests/svg/custom/tref-own-content-removal.svg: Removed.
* LayoutTests/svg/custom/tref-remove-target-crash-expected.svg: Removed.
* LayoutTests/svg/custom/tref-remove-target-crash.svg: Removed.
* LayoutTests/svg/custom/tref-shadowdom-expected.txt: Removed.
* LayoutTests/svg/custom/tref-shadowdom.html: Removed.
* LayoutTests/svg/custom/tref-stale-listener-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-stale-listener-crash.html: Removed.
* LayoutTests/svg/custom/tref-update.svg: Removed.
* LayoutTests/svg/custom/tref-with-progress-tag-setpseudo-assert-expected.txt: Removed.
* LayoutTests/svg/custom/tref-with-progress-tag-setpseudo-assert.html: Removed.
* LayoutTests/svg/custom/use-disallowed-element-clear-corresponding-element-expected.txt: Removed.
* LayoutTests/svg/custom/use-disallowed-element-clear-corresponding-element.html: Removed.
* LayoutTests/svg/custom/use-tref-crash-expected.txt: Removed.
* LayoutTests/svg/custom/use-tref-crash.svg: Removed.
* LayoutTests/svg/dom/svg-use-infinite-loop-cloning.html:
* LayoutTests/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.txt: Removed.
* LayoutTests/svg/dynamic-updates/SVGTRefElement-dom-href-attr.html: Removed.
* LayoutTests/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/svg/foreignObject/text-tref-02-b.svg: Removed.
* LayoutTests/svg/text/text-align-04-b.svg:
* LayoutTests/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/svg/text/text-tref-01-b.svg: Removed.
* LayoutTests/svg/text/tref-display-inherit-expected.txt: Removed.
* LayoutTests/svg/text/tref-display-inherit.html: Removed.
* LayoutTests/svg/text/tref-event-listener-crash-expected.txt: Removed.
* LayoutTests/svg/text/tref-event-listener-crash.svg: Removed.
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/dom/EventListener.h:
* Source/WebCore/dom/Text.cpp:
(WebCore::Text::createTextRenderer):
(WebCore::isSVGShadowText): Deleted.
* Source/WebCore/rendering/svg/legacy/SVGResources.cpp:
(WebCore::clipperFilterMaskerTags):
(WebCore::fillAndStrokeTags):
* Source/WebCore/svg/SVGElement.cpp:
(WebCore::isSVGLayerAwareElement):
(WebCore::SVGElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTRefElement.cpp: Removed.
* Source/WebCore/svg/SVGTRefElement.h: Removed.
* Source/WebCore/svg/SVGTRefElement.idl: Removed.
* Source/WebCore/svg/SVGTSpanElement.cpp:
(WebCore::SVGTSpanElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextElement.cpp:
(WebCore::SVGTextElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextPathElement.cpp:
(WebCore::SVGTextPathElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextPositioningElement.cpp:
(WebCore::SVGTextPositioningElement::elementFromRenderer):
* Source/WebCore/svg/SVGUseElement.cpp:
(WebCore::isDisallowedElement):
* Source/WebCore/svg/svgtags.in:
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c388503a416e404807f4eba1a3c2ad131c003bc7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/53384 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/32736 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/5883 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/56664 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/4110 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/55690 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/40159 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/3877 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/43286 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/2698 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/55482 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/30923 "Found 2 new test failures: http/tests/site-isolation/draw-after-cross-origin-navigation-between-two-remote-frames.html, svg/custom/text-linking.svg (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/46122 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/24418 "Passed tests") | 
| | [❌ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/27802 "Exiting early after 10 failures. 140 tests run. 1 failures") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/3447 "Found 2 new test failures: svg/custom/text-linking.svg, svg/text/text-align-04-b.svg (failure)") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/2266 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/49563 "Passed tests") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/3614 "Found 2 new test failures: svg/custom/text-linking.svg, svg/text/text-align-04-b.svg (failure)") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/58259 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/28535 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/3609 "Found 2 new test failures: svg/custom/text-linking.svg, svg/text/text-align-04-b.svg (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/50687 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/29748 "Built successfully") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/46324 "Exiting early after 10 failures. 182 tests run. 2 failures") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/50020 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/30672 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/29515 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rwlbuis commented on LayoutTests/imported/w3c/web-platform-tests/accessibility/crashtests/serialize-with-no-document.html:
```diff
@@ -1,4 +1,4 @@
-><a>feColorMatrix><div id="container"><0<script>0000A00AAAAA00A<tspan></tspan>000<do>AA0AA0<fieldset></fieldset>A00AA<font><feComponentTransfer></feComponentTransfer><insertedLegend>AAA0A0A0A00A<img>0<link>0A00A0AAAAAA0AAA000A<feDiffuseLighting><mfrac></mfrac><anchor></anchor>00A0<tspan><tspan><tspan><dcol></dcol><missing_glyph></missing_glyph><col></col>0A0AAA0AA0AA0A000A0<mover></mover><form></ ...
+><a>feColorMatrix><div id="container"><0<script>0000A00AAAAA00A<tspan></tspan>000<do>AA0AA0<fieldset></fieldset>A00AA<font><feComponentTransfer></feComponentTransfer><insertedLegend>AAA0A0A0A00A<img>0<link>0A00A0AAAAAA0AAA000A<feDiffuseLighting><mfrac></mfrac><anchor></anchor>00A0<tspan><tspan><tspan><dcol></dcol><missing_glyph></missing_glyph><col></col>0A0AAA0AA0AA0A000A0<mover></mover><form></ ...
```
I don't think we want local changes against upstream WPT repo; at any time we just want to sync SVG WPT tests with whatever changes are there and not worry about differences/conflicts.

lukewarlow replied:
I've exported this to a PR there so it will remain in sync.

rwlbuis commented on LayoutTests/svg/animations/animate-update-crash.xhtml:
```diff
@@ -1,20 +0,0 @@
-                    <tref xlink:href="#foo">
-                        <animateColor attributeName="keyPoints"></animateColor>
-                        <animateColor attributeName="xlink:href"></animateColor>
-                    </tref>
```
This is likely still a valid crash test, since tref does not seem to be the focus here but animation is. It is likely enough to substitute tref for another svg tag here (maybe <a> if it needs to have explicit xlink:href support).

rwlbuis commented on LayoutTests/svg/custom/use-disallowed-element-clear-corresponding-element.html:
```diff
@@ -1,25 +0,0 @@
-    </svg>
-    <shadow id="shadow">
-        <svg>
-            <tref href="#svg-pattern-2"/>
```
Possibly this test does not rely on tref exclusively.

lukewarlow committed:
Remove support for SVGTRefElement
https://bugs.webkit.org/show_bug.cgi?id=246191

Reviewed by NOBODY (OOPS!).

This patch removes SVGTRefElement and associated tests.

WebKit is the only engine that supports it, and it's not part of the SVG spec anymore.

* LayoutTests/fast/block/child-not-removed-from-parent-lineboxes-crash.html:
* LayoutTests/fast/rendering/nested-render-tree-update-crash-expected.txt: Removed.
* LayoutTests/fast/rendering/nested-render-tree-update-crash.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-change-in-shadow-tree-expected.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-change-in-shadow-tree.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-in-shadow-tree-expected.html: Removed.
* LayoutTests/fast/shadow-dom/svg-thref-href-in-shadow-tree.html: Removed.
* LayoutTests/http/tests/svg/resources/svg-tref.svg: Removed.
* LayoutTests/http/tests/svg/tref-adoptNode-crash-expected.txt: Removed.
* LayoutTests/http/tests/svg/tref-adoptNode-crash.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/accessibility/crashtests/serialize-with-no-document.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/idlharness.window.js:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/tools/third_party/html5lib/html5lib/filters/sanitizer.py:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/glib/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/glib/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/glib/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/glib/svg/custom/text-linking-expected.txt:
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/glib/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/glib/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/glib/svg/text/text-align-04-b-expected.txt:
* LayoutTests/platform/glib/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/gtk/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/gtk/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/gtk/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/gtk/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.png: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/ios/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/ios/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/ios/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/ios/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/ios/svg/text/text-align-04-b-expected.txt:
* LayoutTests/platform/ios/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/ios/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/styling-pres-02-f-manual-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-align-04-b-manual-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-01-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-02-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/import/text-tref-03-b-manual-expected.txt: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/batik/text/textProperties-expected.txt:
* LayoutTests/platform/mac/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/mac/svg/custom/broken-internal-references-expected.txt:
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-dom-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-referenced-element-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-tref-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/tref-own-content-removal-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/mac/svg/custom/tref-update-expected.png: Removed.
* LayoutTests/platform/mac/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/mac/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.png: Removed.
* LayoutTests/platform/mac/svg/foreignObject/text-tref-02-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/platform/mac/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/mac/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-change-href-dom-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-change-href-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-referenced-element-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/text-tref-03-b-tref-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/tref-own-content-removal-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/custom/tref-update-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1-SE/text-tref-03-b-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-align-04-b-expected.txt:
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-tref-01-b-expected.png: Removed.
* LayoutTests/platform/wpe/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/text/text-tref-01-b-expected.png: Removed.
* LayoutTests/svg/W3C-SVG-1.1/resources/text-align-02-f.svg:
* LayoutTests/svg/W3C-SVG-1.1/resources/text-extTref-BE-18-targ.svg: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-align-04-b.svg:
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b-expected.txt: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b-w3c.png: Removed.
* LayoutTests/svg/W3C-SVG-1.1/text-tref-01-b.svg: Removed.
* LayoutTests/svg/animations/animate-update-crash-expected.txt: Removed.
* LayoutTests/svg/animations/animate-update-crash.xhtml: Removed.
* LayoutTests/svg/batik/text/textProperties-expected.txt:
* LayoutTests/svg/batik/text/textProperties.svg:
* LayoutTests/svg/batik/text/textStyles.svg:
* LayoutTests/svg/custom/broken-internal-references.svg:
* LayoutTests/svg/custom/bug78700-expected.txt: Removed.
* LayoutTests/svg/custom/bug78700.svg: Removed.
* LayoutTests/svg/custom/disallowed-text-content-rendering.svg:
* LayoutTests/svg/custom/global-constructors-expected.txt:
* LayoutTests/svg/custom/global-constructors.html:
* LayoutTests/svg/custom/invalid-ctm.svg:
* LayoutTests/svg/custom/invalid-text-content-expected.txt:
* LayoutTests/svg/custom/invalid-text-content.svg:
* LayoutTests/svg/custom/remove-subtree-including-text-with-duplicate-id-referenced-by-later-tref-expected.txt: Removed.
* LayoutTests/svg/custom/remove-subtree-including-text-with-duplicate-id-referenced-by-later-tref.html: Removed.
* LayoutTests/svg/custom/reparent-tref-element-expected.txt: Removed.
* LayoutTests/svg/custom/reparent-tref-element.html: Removed.
* LayoutTests/svg/custom/text-linking.svg:
* LayoutTests/svg/custom/text-tref-03-b-change-href-dom.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-change-href.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic-expected.txt: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic2-expected.txt: Removed.
* LayoutTests/svg/custom/text-tref-03-b-dynamic2.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-referenced-element-removal.svg: Removed.
* LayoutTests/svg/custom/text-tref-03-b-tref-removal.svg: Removed.
* LayoutTests/svg/custom/tref-clone-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-clone-crash.html: Removed.
* LayoutTests/svg/custom/tref-nested-events-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-nested-events-crash.svg: Removed.
* LayoutTests/svg/custom/tref-own-content-removal.svg: Removed.
* LayoutTests/svg/custom/tref-remove-target-crash-expected.svg: Removed.
* LayoutTests/svg/custom/tref-remove-target-crash.svg: Removed.
* LayoutTests/svg/custom/tref-shadowdom-expected.txt: Removed.
* LayoutTests/svg/custom/tref-shadowdom.html: Removed.
* LayoutTests/svg/custom/tref-stale-listener-crash-expected.txt: Removed.
* LayoutTests/svg/custom/tref-stale-listener-crash.html: Removed.
* LayoutTests/svg/custom/tref-update.svg: Removed.
* LayoutTests/svg/custom/tref-with-progress-tag-setpseudo-assert-expected.txt: Removed.
* LayoutTests/svg/custom/tref-with-progress-tag-setpseudo-assert.html: Removed.
* LayoutTests/svg/custom/use-disallowed-element-clear-corresponding-element-expected.txt: Removed.
* LayoutTests/svg/custom/use-disallowed-element-clear-corresponding-element.html: Removed.
* LayoutTests/svg/custom/use-tref-crash-expected.txt: Removed.
* LayoutTests/svg/custom/use-tref-crash.svg: Removed.
* LayoutTests/svg/dom/svg-use-infinite-loop-cloning.html:
* LayoutTests/svg/dynamic-updates/SVGTRefElement-dom-href-attr-expected.txt: Removed.
* LayoutTests/svg/dynamic-updates/SVGTRefElement-dom-href-attr.html: Removed.
* LayoutTests/svg/foreignObject/text-tref-02-b-expected.txt: Removed.
* LayoutTests/svg/foreignObject/text-tref-02-b.svg: Removed.
* LayoutTests/svg/text/text-align-04-b.svg:
* LayoutTests/svg/text/text-tref-01-b-expected.txt: Removed.
* LayoutTests/svg/text/text-tref-01-b.svg: Removed.
* LayoutTests/svg/text/tref-display-inherit-expected.txt: Removed.
* LayoutTests/svg/text/tref-display-inherit.html: Removed.
* LayoutTests/svg/text/tref-event-listener-crash-expected.txt: Removed.
* LayoutTests/svg/text/tref-event-listener-crash.svg: Removed.
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/dom/EventListener.h:
* Source/WebCore/dom/Text.cpp:
(WebCore::Text::createTextRenderer):
(WebCore::isSVGShadowText): Deleted.
* Source/WebCore/rendering/svg/legacy/SVGResources.cpp:
(WebCore::clipperFilterMaskerTags):
(WebCore::fillAndStrokeTags):
* Source/WebCore/svg/SVGElement.cpp:
(WebCore::isSVGLayerAwareElement):
(WebCore::SVGElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTRefElement.cpp: Removed.
* Source/WebCore/svg/SVGTRefElement.h: Removed.
* Source/WebCore/svg/SVGTRefElement.idl: Removed.
* Source/WebCore/svg/SVGTSpanElement.cpp:
(WebCore::SVGTSpanElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextElement.cpp:
(WebCore::SVGTextElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextPathElement.cpp:
(WebCore::SVGTextPathElement::childShouldCreateRenderer const):
* Source/WebCore/svg/SVGTextPositioningElement.cpp:
(WebCore::SVGTextPositioningElement::elementFromRenderer):
* Source/WebCore/svg/SVGUseElement.cpp:
(WebCore::isDisallowedElement):
* Source/WebCore/svg/svgtags.in:
